check.sh

```diff
@@ -62,6 +62,10 @@ mypy trio --show-error-end --platform darwin | python ./trio/_tools/mypy_annotat
     || { echo "* Mypy (Mac) found type errors." >> $GITHUB_STEP_SUMMARY; MYPY=1; }
 mypy trio --show-error-end --platform win32 | python ./trio/_tools/mypy_annotate.py --dumpfile mypy_annotate.dat --platform Windows \
     || { echo "* Mypy (Windows) found type errors." >> $GITHUB_STEP_SUMMARY; MYPY=1; }
+
+# for now, just run interface typing checks on the native platform...
+mypy trio/_tests/type_tests --show-error-end | python ./trio/_tools/mypy_annotate.py --dumpfile mypy_annotate.dat --platform Interface \
+    || { echo "* Mypy (Interface) found type errors." >> $GITHUB_STEP_SUMMARY; MYPY=1; }
```

Comment:
Reading this made me think `Interface` was a magic string, so would probably be better to leave it out.

But isn't this fully covered by the above mypy runs? `trio/_tests/type_tests` shouldn't be excluded or anything in `pyproject.toml` AFAICT.

Reply:
Oh oops, I forgot that it would be covered. Yeah that makes sense.

Reply:
93e7d80 fixes this